### DISCUSSION TITLE
blocking access to my searxng domain

### DISCUSSION THREAD

**Discussion by lannic** [searxng/searxng#4086] [Q&A]
Hi everyone

I have a rapsberry pi with a sub domain for searxng.
But sometimes, all my domain name access is blocked by too much request.
A bit of log int the attach file
[access.log](https://github.com/user-attachments/files/17980283/access.log)

I enable the limiter in the limiter.toml file : `link_token = true`
and in the settings.yml : `limiter: true`
but it doesn't work.

**Comment by return42** [comment]
Check your proxy setup --> read https://github.com/searxng/searxng/issues/3191#issuecomment-2202729527 and the following comments.

-----


For a lively discussion its recommended to participate from the SearXNG community at [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org).

**Comment by unixfox** [comment]
You don't need to enable limiter if it's only for private usage.

**Comment by andersnordh** [comment]
Hi "lannic", some of my thoughts and the need for a proper firewall you can find here ...
https://github.com/searxng/searxng/discussions/4068#discussion-7573757
Have a nice evening,
//Anders
